linkcheck/htmlutil/linkparse.py

```diff
@@ -175,10 +175,10 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
-        if tag =="a" and attrs.get_true('class', u''):
-           if ("broken_link" in attrs.get('class')):
-               log.debug(LOG_CHECK," Found a broken_link to %s, not considering further", attrs.get('href'))
-               return
+         if tag == "a" and attrs.get('class'):
```

Comment:
The indentation looks wrong on this line (it's not matching the previous `if` line).  I would expect a syntax error from the Python parser.

Reply:
or at least linting... which our newer CI should at least flag...

Reply:
I think this fixed in later commits. (My editor keeps defaulting back to tab =tab, not tab = 4 spaces)

Reply:
ah sorry for the noise